infrastructure/helm/hacks/apiserver-patch.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
This change is being added in #235 and is related with `kuberay-api-server` so I think it can be removed from the PR. Correct me if I'm wrong @akihikokuroda .